lib/ansible/modules/cloud/amazon/ecs_task.py

```diff
@@ -243,14 +253,21 @@ def run_task(self, cluster, task_definition, overrides, count, startedBy, launch
             params['networkConfiguration'] = self.format_network_configuration(self.module.params['network_configuration'])
         if launch_type:
             params['launchType'] = launch_type
+        if tags:
+            if (self.ecs_task_long_format_enabled()):
+                params['tags'] = ansible_dict_to_boto3_tag_list(tags, 'key', 'value')
+            else:
+                self.module.fail_json(msg="Account does not support long format task arns")
```

Comment:
I get that long format task ARNS might be a prerequisite for tags but might be better say e.g.

> "Cannot set task tags: long format task arns are required to set tags"